analyzer.go

```diff
@@ -251,6 +251,10 @@ func (a *Analyzer) getImageIdentifier(image imgutil.Image) (string, error) {
 		a.Logger.Infof("Image with name %q not found", image.Name())
 		return "", nil
 	}
+	if !image.Valid() {
+		a.Logger.Infof("Ignoring image with name %q because it was corrupt", image.Name())
```

Comment:
Is a `corrupt` image the only reason it would fail at this point?

Reply:
Yes, `Valid()` will return false if the image is not found, the manifest cannot be parsed or something listed in the manifest does not exist in the registry. The `!image.Found()` check on line 254, has already confirmed that manifest exists, so this condition will only occur if the something is corrupt.

Reply:
This wasn't actually preventing the tag from being decoded.